src/Controller/CustomerController.php

```diff
@@ -44,14 +44,14 @@ public function addCustomer(Request $request): JsonResponse
 
          }
 
-         $this->customerRepository->saveCustomer($firstName, $lastName, $email, $phoneNumber);
-         return new JsonResponse(['status' => "Customer created!"], Response::HTTP_CREATED);
+         $id = $this->customerRepository->saveCustomer($firstName, $lastName, $email, $phoneNumber);
+         return new JsonResponse(['status' => "Customer created with id number {$id}"], Response::HTTP_CREATED);
```

Comment:
`status` might not be a very good choice for naming the response message key. 
Maybe `message` would suffice?

Same is applicable for all usages in this file.

Reply:
naming response changed from `status` to `message`